.bumpversion.cfg

```diff
@@ -10,6 +10,10 @@ serialize =
 
 [bumpversion:file:.env]
 
+[bumpversion:file:.env.dev]
+
 [bumpversion:file:kube/overlays/stable/.env]
 
 [bumpversion:file:kube/overlays/stable/kustomization.yaml]
+
+[bumpversion:file:airbyte-db/src/main/resources/schema.sql]
```

Comment:
how does this work? does it just find the version string in these files and replace it? nifty, but spooky.

Reply:
Yes, the default behavior is to search for the current version string:
![Screenshot 2021-01-20 at 19 23 25](https://user-images.githubusercontent.com/7718175/105218190-2372b580-5b55-11eb-9908-981987f5fa5f.png)

but you could change it to match more things around too, for example if you want to replace this variable but not all other mentions of the current version in the same file:

```
[bumpversion:file:src/__init__.py]
search = __version__ = '{current_version}'
replace = __version__ = '{new_version}'
```

Reply:
cool.